src/Compilers/CSharp/Portable/CSharpParseOptions.cs

```diff
@@ -100,18 +104,19 @@ public override IEnumerable<string> PreprocessorSymbolNames
         }
 
         // No validation
-        internal CSharpParseOptions(
+        private CSharpParseOptions(
```

Comment:
@AlekseyTs sure I can. but I guess I am okay to checkin after the change? @srivatsn ?